resources/shared/helpers.mjs

```diff
@@ -24,6 +24,7 @@ export function getAllElements(selector, path = [], lookupStartNode = document)
 }
 
 export function forceLayout() {
+    // FIXME: sync implementation with Page.prototype.layout().
```

Comment:
Why not doing the sync now?

Reply:
ah I see, it's used in the remote page.
Do you have a plan about this?

Reply:
I would like to generally get rid of the Page abstraction and just have the helpers there (but probably it's easier to move the Page to the shared module).

Reply:
ah let me just move all code to this helper for now... then at least it's uniform :)